demos/python_demos/human_pose_estimation_3d_demo/human_pose_estimation_3d_demo.py

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env python
+"""
+ Copyright (c) 2019 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import json
+import os
+
+import cv2
+from argparse import ArgumentParser, SUPPRESS
+import numpy as np
+
+from modules.inference_engine import InferenceEngine
+from modules.input_reader import InputReader
+from modules.draw import Plotter3d, draw_poses
+from modules.parse_poses import parse_poses
+
+
+def rotate_poses(poses_3d, R, t):
+    R_inv = np.linalg.inv(R)
+    for pose_id in range(len(poses_3d)):
+        pose_3d = poses_3d[pose_id].reshape((-1, 4)).transpose()
+        pose_3d[0:3, :] = np.dot(R_inv, pose_3d[0:3, :] - t)
+        poses_3d[pose_id] = pose_3d.transpose().reshape(-1)
+
+    return poses_3d
+
+
+if __name__ == '__main__':
+    parser = ArgumentParser(description='Lightweight 3d human pose estimation demo. '
+                                        'Press esc to exit, "p" to (un)pause video or process next image.',
+                            add_help=False)
+    args = parser.add_argument_group('Options')
+    args.add_argument('-h', '--help', action='help', default=SUPPRESS,
+                      help='Show this help message and exit.')
+    args.add_argument('-m', '--model',
+                      help='Required. Path to an .xml file with a trained model '
+                           '(or .pth in case of PyTorch inference).',
```

Comment:
Please, remove PyTorch mentions since it is not used anymore. The same is for `README.md`.